src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Ball extends Lottery {
+
+    private String color;
+    private int number;
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public String toString() {
+        return "Random color is: " + color + "\n" + "Random number is: " + number;
```

Comment:
```suggestion
        return "color is: " + color + ", " + " number is: " + number;
```
what if we want to use this class in another context?

Reply:
fixed